api.md

```diff
@@ -202,7 +202,8 @@ function.
 
 | Name              | Description                                   |
 |-------------------|-----------------------------------------------|
-| `if_not_attached` | Prevents error if data node is already attached to the hypertable. A notice will be printed that the data node is attached. |
+| `if_not_attached` | Prevents error if data node is already attached to the hypertable. A notice will be printed that the data node is attached. Defaults to `FALSE`. |
+| `repartition`     | Expand the number of partitions in the space dimension to the number of data nodes after attaching this data node. Defaults to `TRUE`. |
```

Comment:
Maybe we should say something like:

```
Allow updates to the hypertable's partitioning configuration to ensure the attached data node will be utilized.
```

This gives more of the _why_ as opposed to the _how_.

Reply:
Done, approximately. PTAL.